lstchain/reco/r0_to_dl1.py

```diff
@@ -569,6 +616,17 @@ def r0_to_dl1(
                     containers=[event.index, dl1_tel, extra_im]
                 )
 
+                if lhfit_fitter is not None:
+                    lhfit_container = apply_lh_fit(event, telescope_id, dl1_container, lhfit_fitter)
+                    # Plotting code for development purpose only, will disappear in final release
+                    if lhfit_fitter.verbose >= 2 and lhfit_container["lhfit_call_status"] == 1:
+                        plot_debug(lhfit_fitter, event, telescope_id, dl1_container, str(event.index.event_id))
```

Comment:
is this `plot_debug` really necessary? it could break some tests in the future just because of including this plotting here.